snorkel/labeling/model/logger.py

```diff
@@ -50,7 +50,7 @@ def log(self, metrics_dict: Dict[str, float]) -> None:
         -------
         >>> logger = Logger(log_train_every=5)
         >>> metrics_dict = {"train/loss": 5.00}
-        >>> logger.log(metrics_dict)
+        >>> if logger.check(): logger.log(metrics_dict)
```

Comment:
This is extremely bizarre. Not an example usage. Looks like we hacked this in to pass a doctest.

Reply:
`logger.log()` directly reports `[-1 epochs]:`, which is odd, but it will never be called in the wild without `logger.check()` being called first. We can simplify the doctest case; they'll just see a printout that they shouldn't ever see in real usage.